From b71cb53bc9bfa6cf613ff651c707db3e097c3593 Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Fri, 5 Jan 2018 17:40:58 -0800
Subject: [PATCH 05/10] dma: xilinx: dpdma: Fix a comment

This patch comes from:
  https://github.com/Xilinx/linux-xlnx.git

Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
(cherry picked from commit 8104905467fdd459a1abc54826b08815b84f24b3)
---
 drivers/dma/xilinx/xilinx_dpdma.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/dma/xilinx/xilinx_dpdma.c b/drivers/dma/xilinx/xilinx_dpdma.c
index 6a3e185..3fcd276 100644
--- a/drivers/dma/xilinx/xilinx_dpdma.c
+++ b/drivers/dma/xilinx/xilinx_dpdma.c
@@ -263,7 +263,7 @@ enum xilinx_dpdma_chan_status {
  * Transition is triggered as following:
  *
  * -> allocated_desc : a descriptor allocation
- * allocated_desc -> submitted_desc: a descriptorsubmission
+ * allocated_desc -> submitted_desc: a descriptor submission
  * submitted_desc -> pending_desc: request to issue pending a descriptor
  * pending_desc -> active_desc: VSYNC intr when a desc is scheduled to DPDMA
  * active_desc -> done_list: VSYNC intr when DPDMA switches to a new desc
-- 
1.8.5.2.233.g932f7e4

